plugins/inputs/neptune_apex/README.md

```diff
@@ -0,0 +1,145 @@
+# neptune_apex Input Plugin
+
+The neptune_apex Input Plugin collects real-time data from the Apex's status.xml page.
+
+### Configuration
+
+```toml
+[[inputs.neptune_apex]]
+  ## The Neptune Apex plugin reads the publicly available status.xml data from a local Apex.
+  ## Measurements will be logged under "apex".
+
+  ## The hostname/IP of the local Apex(es). If you specify more than one server, they will
+  ## be differentiated by the "hostname" tag.
+  servers = [
+    "apex.local",
```

Comment:
Let's have this take a base url in case we want to add support for TLS later.  Even though the api is over HTTP, its possible that a user will want to use a proxy to access the server.

Reply:
Done.